prepare_venv.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-python3 -m virtualenv venv
-source venv/bin/activate
+python3 -m venv virtualenv
```

Comment:
I don't understand why you did that. `venv` and `virtualenv` a different packages. they are doing mostly the same, `venv` is prebuilt but `virtualenv` more preferable to use and `venv` is more preferable name for virtual environment. if you had trouble with running that scripts please try:
```bash
pip install virtualenv
```
and please change it as it was before.